python/python/glide/async_commands/core.py

```diff
@@ -3128,6 +3128,127 @@ async def xpending_range(
             await self._execute_command(RequestType.XPending, args),
         )
 
+    async def xautoclaim(
+        self,
+        key: str,
+        group_name: str,
+        consumer_name: str,
+        min_idle_time_ms: int,
+        start: str,
+        count: Optional[int] = None,
+    ) -> List[Union[str, Mapping[str, List[List[str]]], List[str]]]:
+        """
+        Transfers ownership of pending stream entries that match the specified criteria.
+
+        See https://valkey.io/commands/xautoclaim for more details.
+
+        Args:
+            key (str): The key of the stream.
+            group_name (str): The consumer group name.
+            consumer_name (str): The consumer name.
+            min_idle_time_ms (int): Filters the claimed entries to those that have been idle for more than the specified
+                value.
+            start (str): Filters the claimed entries to those that have an ID equal or greater than the specified value.
+            count (Optional[int]): Limits the number of claimed entries to the specified value.
+
+        Returns:
+            List[Union[str, Mapping[str, List[List[str]]], List[str]]]: A list containing the following elements:
+                - A stream ID to be used as the start argument for the next call to `XAUTOCLAIM`. This ID is equivalent
+                to the next ID in the stream after the entries that were scanned, or "0-0" if the entire stream was
+                scanned.
+                - A mapping of the claimed entries, with the keys being the claimed entry IDs and the values being a
+                2D list of the field-value pairs in the format `[[field1, value1], [field2, value2], ...]`.
+                - If you are using Redis 7.0.0 or above, the response list will also include a list containing the
+                message IDs that were in the Pending Entries List but no longer exist in the stream. These IDs are
+                deleted from the Pending Entries List.
+
+        Examples:
+            # Redis version < 7.0.0:
+            >>> await client.xautoclaim("my_stream", "my_group", "my_consumer", 3_600_000, "0-0")
+                ["0-0", {"1-1": [["field1", "value1"], ["field2", "value2"]]}]
```

Comment:
Can you make output multiline, like we have in https://github.com/aws/glide-for-redis/pull/1716?